internal/tools/orchestrator/scheduler/instanceinfo/instance.go

```diff
@@ -161,7 +164,7 @@ func (r *InstanceReader) Limit(n int) *InstanceReader {
 }
 func (r *InstanceReader) Do() ([]InstanceInfo, error) {
 	instanceinfo := []InstanceInfo{}
-	expr := r.db.Where(strutil.Join(r.conditions, " AND ", true)).Order("started_at desc")
+	expr := r.db.Where(strutil.Join(r.conditions, " AND ", true), r.values[0], r.values[1]).Order("started_at desc")
```

Comment:
other conditions `runtime` `service` may be not have values, so values[0], values[idx] and etc will cause panic